tests/test_transformer.py

```diff
@@ -0,0 +1,142 @@
+#    limitations under the License.
+
+import unittest
+import unittest.mock
+import warnings
+
+import dimod
+import numpy as np
+
+from dwave.cloud.exceptions import ConfigFileError, SolverAuthenticationError
+from dwave.system import LeapHybridCQMSampler
+from sklearn.datasets import load_iris
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.pipeline import Pipeline
+
+from dwave.plugins.sklearn.transformers import SelectFromQuadraticModel
+
+
+class MockCQM(dimod.ExactCQMSolver):
+    def sample_cqm(self, cqm: dimod.CQM, *, time_limit: float) -> dimod.SampleSet:
+        return super().sample_cqm(cqm)
+
+    def min_time_limit(self, cqm):
+        return 1
+
+
+@unittest.mock.patch("dwave.plugins.sklearn.transformers.LeapHybridCQMSampler", MockCQM)
+class TestSelectFromQuadraticModel(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        rng = np.random.default_rng(138984)
+        cls.X = rng.uniform(-10, 10, size=(100, 9))
+        cls.y = np.asarray(rng.uniform(0, 1, size=100) > 0.5, dtype=int)
+
+    def test_init_good(self):
+        a = SelectFromQuadraticModel()
+
+        b = SelectFromQuadraticModel(alpha=0.1)
+
+        c = SelectFromQuadraticModel(alpha=0.1, time_limit=30)
+
+        d = SelectFromQuadraticModel(time_limit=15)
+
+        self.assertIsInstance(a, SelectFromQuadraticModel)
+        self.assertIsInstance(b, SelectFromQuadraticModel)
+        self.assertIsInstance(c, SelectFromQuadraticModel)
+        self.assertIsInstance(d, SelectFromQuadraticModel)
+
+        self.assertEqual(a._alpha, 0.5)
+        self.assertEqual(b._alpha, 0.1)
+        self.assertEqual(c._alpha, 0.1)
+        self.assertEqual(d._alpha, 0.5)
+
+        self.assertEqual(a._time_limit, None)
+        self.assertEqual(b._time_limit, None)
+        self.assertEqual(c._time_limit, 30)
+        self.assertEqual(d._time_limit, 15)
+
+        self.assertIsInstance(
+            SelectFromQuadraticModel(alpha=0), SelectFromQuadraticModel
+        )
+
+    def test_init_bad(self):
+        self.assertRaises(ValueError, SelectFromQuadraticModel, alpha=-10)
+        self.assertRaises(ValueError, SelectFromQuadraticModel, alpha=10)
+
+    def test_fit(self):
+        selector = SelectFromQuadraticModel(num_features=7)
+
+        # test default numpy
+
+        selector.fit(self.X, self.y)
+        self.assertEqual(sum(selector._mask), 7)
+
+        try:
+            self.X[:, selector._mask]
+        except Exception as e:
+            self.fail(e)
+
+        # test non-default numpy
+
+        selector.fit(self.X, self.y, num_features=5)
+        self.assertEqual(sum(selector._mask), 5)
+
+        try:
+            self.X[:, selector._mask]
+        except Exception as e:
+            self.fail(e)
+
+    def test_fit_transform(self):
+        selector = SelectFromQuadraticModel(num_features=7)
+
+        # test numpy without fit
+        x = selector.fit_transform(self.X, self.y, num_features=5)
```

Comment:
I no longer see a `fit_transform` method

Reply:
Indeed. [It is inherited](https://github.com/scikit-learn/scikit-learn/blob/9aaed498795f68e5956ea762fef9c440ca9eb239/sklearn/base.py#L850).